### ISSUE TITLE
Previewed image is displayed too small

### ISSUE THREAD

**Issue by vivadavid** [upscayl/upscayl#455]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.
- [X] I have tried [setting my GPU ID](https://github.com/upscayl/upscayl/wiki/Guide#gpu-id).

### Paste the logs

```sh
⚙️ Getting saveImageAs from localStorage png
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🎯 GET_MODELS_LIST:  
🔄 Resetting image paths
🔀 Setting model to realesrgan-x4plus
🔄 Resetting image paths
⤵️ Dropped file:  {"type":"image/jpeg","filePath":"C:\\Users\\vivad\\Documents\\F5KeobuWwAAbAPc.jpg","extension":"jpg"}
🖼 Setting image path:  C:\Users\vivad\Documents\F5KeobuWwAAbAPc.jpg
🗂 Setting output path:  C:\Users\vivad\Documents
🖼 imagePath:  C:\Users\vivad\Documents\F5KeobuWwAAbAPc.jpg
🔤 Extension:  jpg
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🐞 BACKEND REPORTED:  📢 Upscayl Command:  -i,C:\Users\vivad\Documents\F5KeobuWwAAbAPc.jpg,-o,C:\Users\vivad\Documents\F5KeobuWwAAbAPc_upscayl_4x_realesrgan-x4plus.png,-s,4,-m,C:\Program Files\Upscayl\resources\models,-n,realesrgan-x4plus,,,-f,png
🐞 BACKEND REPORTED:  image upscayl:  [0 NVIDIA GeForce RTX 2060]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 2060]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 2060]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 2060]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1
[1 AMD Radeon(TM) Graphics]  queueC=1[2]  queueG=0[1]  queueT=2[1]
[1 AMD Radeon(TM) Graphics]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 AMD Radeon(TM) Graphics]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[1 AMD Radeon(TM) Graphics]  subgroup=64  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 NVIDIA GeForce RTX 2060]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 2060]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 2060]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 2060]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1
[1 AMD Radeon(TM) Graphics]  queueC=1[2]  queueG=0[1]  queueT=2[1]
[1 AMD Radeon(TM) Graphics]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 AMD Radeon(TM) Graphics]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[1 AMD Radeon(TM) Graphics]  subgroup=64  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  0,00%

🚧 UPSCAYL_PROGRESS:  0,00%

🐞 BACKEND REPORTED:  image upscayl:  2,86%

🚧 UPSCAYL_PROGRESS:  2,86%

🐞 BACKEND REPORTED:  image upscayl:  5,71%

🚧 UPSCAYL_PROGRESS:  5,71%

🐞 BACKEND REPORTED:  image upscayl:  8,57%

🚧 UPSCAYL_PROGRESS:  8,57%

🐞 BACKEND REPORTED:  image upscayl:  11,43%

🚧 UPSCAYL_PROGRESS:  11,43%

🐞 BACKEND REPORTED:  image upscayl:  14,29%

🚧 UPSCAYL_PROGRESS:  14,29%

🐞 BACKEND REPORTED:  image upscayl:  17,14%

🚧 UPSCAYL_PROGRESS:  17,14%

🐞 BACKEND REPORTED:  image upscayl:  20,00%

🚧 UPSCAYL_PROGRESS:  20,00%

🐞 BACKEND REPORTED:  image upscayl:  22,86%

🚧 UPSCAYL_PROGRESS:  22,86%

🐞 BACKEND REPORTED:  image upscayl:  25,71%

🚧 UPSCAYL_PROGRESS:  25,71%

🐞 BACKEND REPORTED:  image upscayl:  28,57%

🚧 UPSCAYL_PROGRESS:  28,57%

🐞 BACKEND REPORTED:  image upscayl:  31,43%

🚧 UPSCAYL_PROGRESS:  31,43%

🐞 BACKEND REPORTED:  image upscayl:  34,29%

🚧 UPSCAYL_PROGRESS:  34,29%

🐞 BACKEND REPORTED:  image upscayl:  37,14%

🚧 UPSCAYL_PROGRESS:  37,14%

🐞 BACKEND REPORTED:  image upscayl:  40,00%

🚧 UPSCAYL_PROGRESS:  40,00%

🐞 BACKEND REPORTED:  image upscayl:  42,86%

🚧 UPSCAYL_PROGRESS:  42,86%

🐞 BACKEND REPORTED:  image upscayl:  45,71%

🚧 UPSCAYL_PROGRESS:  45,71%

🐞 BACKEND REPORTED:  image upscayl:  48,57%

🚧 UPSCAYL_PROGRESS:  48,57%

🐞 BACKEND REPORTED:  image upscayl:  51,43%

🚧 UPSCAYL_PROGRESS:  51,43%

🐞 BACKEND REPORTED:  image upscayl:  54,29%

🚧 UPSCAYL_PROGRESS:  54,29%

🐞 BACKEND REPORTED:  image upscayl:  57,14%

🚧 UPSCAYL_PROGRESS:  57,14%

🐞 BACKEND REPORTED:  image upscayl:  60,00%

🚧 UPSCAYL_PROGRESS:  60,00%

🐞 BACKEND REPORTED:  image upscayl:  62,86%

🚧 UPSCAYL_PROGRESS:  62,86%

🐞 BACKEND REPORTED:  image upscayl:  65,71%

🚧 UPSCAYL_PROGRESS:  65,71%

🐞 BACKEND REPORTED:  image upscayl:  68,57%

🚧 UPSCAYL_PROGRESS:  68,57%

🐞 BACKEND REPORTED:  image upscayl:  71,43%

🚧 UPSCAYL_PROGRESS:  71,43%

🐞 BACKEND REPORTED:  image upscayl:  74,29%

🚧 UPSCAYL_PROGRESS:  74,29%

🐞 BACKEND REPORTED:  image upscayl:  77,14%

🚧 UPSCAYL_PROGRESS:  77,14%

🐞 BACKEND REPORTED:  image upscayl:  80,00%

🚧 UPSCAYL_PROGRESS:  80,00%

🐞 BACKEND REPORTED:  image upscayl:  82,86%

🚧 UPSCAYL_PROGRESS:  82,86%

🐞 BACKEND REPORTED:  image upscayl:  85,71%

🚧 UPSCAYL_PROGRESS:  85,71%

🐞 BACKEND REPORTED:  image upscayl:  88,57%

🚧 UPSCAYL_PROGRESS:  88,57%

🐞 BACKEND REPORTED:  image upscayl:  91,43%

🚧 UPSCAYL_PROGRESS:  91,43%

🐞 BACKEND REPORTED:  image upscayl:  94,29%

🚧 UPSCAYL_PROGRESS:  94,29%

🐞 BACKEND REPORTED:  image upscayl:  97,14%

🚧 UPSCAYL_PROGRESS:  97,14%

🐞 BACKEND REPORTED:  💯 Done upscaling
🐞 BACKEND REPORTED:  ♻ Scaling and converting now...
upscaledImagePath:  C:\Users\vivad\Documents\F5KeobuWwAAbAPc_upscayl_4x_realesrgan-x4plus.png
💯 UPSCAYL_DONE:  C:\Users\vivad\Documents\F5KeobuWwAAbAPc_upscayl_4x_realesrgan-x4plus.png
⚙️ Getting saveImageAs from localStorage png
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Setting gpuId to empty string
⚙️ Getting rememberOutputFolder from localStorage false
```

### Describe the Bug

The previewed image, after applying the upscale, is displayed too small.

### To Reproduce

1. I open an image.
2. I click on _Upscayl_.

### Version/Commit Hash

2.7.5

### OS Specifics

Windows 11 (22H2)

### Expected Behavior

The previewed image should take all the screen, as it does previous to the upscale.

### Screenshots
![Upscayl preview](https://github.com/upscayl/upscayl/assets/9334287/7a8591e6-6eb8-417c-a1d1-1120f242e593)

_No response_

**Comment by NayamAmarshe** [comment]
We're working on a fix for this.

**Comment by vivadavid** [comment]
> We're working on a fix for this.

Thanks!

**Comment by aaronliu0130** [comment]
Pretty sure this is a duplicate of #354.

**Comment by vivadavid** [comment]
> Pretty sure this is a duplicate of #354.

In my case, the upscaled image is automatically and flawlessly saved, though in one particular occasion, the image wasn't saved for some reason. I should have done more testing.

**Comment by NayamAmarshe** [comment]
> > Pretty sure this is a duplicate of #354.
> 
> In my case, the upscaled image is automatically and flawlessly saved, though in one particular occasion, the image wasn't saved for some reason. I should have done more testing.

How large is the upscaled image?

**Comment by vivadavid** [comment]
3876x5380

**Comment by NayamAmarshe** [comment]
> 3876x5380

and what's the file size of the upscaled image?

I think this bug appears when the file size is too large.

**Comment by vivadavid** [comment]
The size depends on the algorithm. I always save as PNG, even if, as it is the case here, the original is a JPEG image.

![filesizes](https://github.com/upscayl/upscayl/assets/9334287/d9357b48-ce1b-481e-bef4-6e3095e3679f)

**Comment by aaronliu0130** [comment]
It seems there was some confusion and the original unrendering issue was confused with basically this issue.

**Comment by NayamAmarshe** [comment]
This is definitely happening because of the large output size. We've added a fix for this in 2.8, need to test if it works.

**Comment by vivadavid** [comment]
> This is definitely happening because of the large output size. We've added a fix for this in 2.8, need to test if it works.

Thank you!

**Comment by NayamAmarshe** [comment]
@vivadavid Could you please help us test this? I can build an EXE for you.

**Comment by vivadavid** [comment]
> @vivadavid Could you please help us test this? I can build an EXE for you.

I've noticed that you've released a new version, but I can test this special build if you want.

**Comment by vivadavid** [comment]
Oh, it's a pre-release and only for Linux. Again, I'll be glad to test your Windows build and provide feedback.

**Comment by vivadavid** [comment]
Hi,

The bug is fixed in version 2.8.1: thanks!

However, at first I was unable to test it. I clicked on _Upscayl_ on the same image I used a few days ago, but nothing happened. I have realized that if there's already a file with the expected name for the upscaled image, no upscale is performed. In my particular case, my original file is _F5KeobuWwAAbAPc.jpg_, I chose _General Photo (real-ersgan)_ and there was already a file called F5KeobuWwAAbAPc_upscayl_4x_realesrgan-x4plus.png. When I deleted it, I was able to rescale my original image.

I can create an issue about it if you want.

In any case, thanks again for 2.8.1!

**Comment by NayamAmarshe** [comment]
> Hi,
> 
> The bug is fixed in version 2.8.1: thanks!
> 
> However, at first I was unable to test it. I clicked on _Upscayl_ on the same image I used a few days ago, but nothing happened. I have realized that if there's already a file with the expected name for the upscaled image, no upscale is performed. In my particular case, my original file is _F5KeobuWwAAbAPc.jpg_, I chose _General Photo (real-ersgan)_ and there was already a file called F5KeobuWwAAbAPc_upscayl_4x_realesrgan-x4plus.png. When I deleted it, I was able to rescale my original image.
> 
> I can create an issue about it if you want.
> 
> In any case, thanks again for 2.8.1!
> 
> 
> 
> 

Awesome!
Yeah, to overwrite pre-existing upscales, you have to enable the overwrite option in the settings :)

Thanks for testing, hope you like it :)

**Comment by vivadavid** [comment]
Sorry, I've just found in _Settings_ an option called _Overwrite previous upscale_: that explains everything.

I wonder if it'd make sense to enable it by default. I think that the expected behaviour, when clicking on _Upscayl_, is for the upscaling to take place, especially when a new version might provide a more refined upscale of the image.
